src/juju/index.js

```diff
@@ -1,13 +1,12 @@
 import Limiter from "async-limiter";
-import jujulib from "@canonical/jujulib";
-// When updating this list of juju facades be sure to update the
-// mangle.reserved list in craco.config.js.
-import annotations from "@canonical/jujulib/api/facades/annotations-v2";
-import client from "@canonical/jujulib/api/facades/client-v2";
-import cloud from "@canonical/jujulib/api/facades/cloud-v3";
-import controller from "@canonical/jujulib/api/facades/controller-v5";
-import modelManager from "@canonical/jujulib/api/facades/model-manager-v5";
-import pinger from "@canonical/jujulib/api/facades/pinger-v1";
+import { connect, connectAndLogin } from "@canonical/jujulib";
+
+import annotations from "@canonical/jujulib/dist/api/facades/annotations-v2";
```

Comment:
Some of these aren't the latest versions but we can update these in follow-ups if we determine that it's necessary.